test/ica.jl

```diff
@@ -49,7 +50,7 @@ C = cov(X, 2)
 
 # FastICA
 
-M = fit(ICA, X, k; do_whiten=false)
+M = fit(ICA, X, k; do_whiten=false, tol=Inf)
```

Comment:
Why change this?

Reply:
Because, the way the cost of the model is currently calculated, it's not guaranteed to converge otherwise. As suggested above I could make another PR to change the calculation to use the dot product (see this commit on my fork:  https://github.com/alexmorley/MultivariateStats.jl/commit/f59b63e742dd3fe8e67a5e338d855d4c0cfe564e) and delete the `tol=Inf` change in the tests in that PR. 

Or I could delete it here and make the change from the above commit in this PR. 

Does that make sense?

Reply:
Yes, please make another PR. Mixing different issues is the recipe not to get a PR merged quickly. :-)

The same applies to the `i = 0` change elsewhere (which BTW introduces a trailing whitespace).

Reply:
OK there's another PR in #45 .